viewer/packages/sector-loader/README.md

```diff
@@ -0,0 +1,54 @@
+# Sector Loader
+The sector loader package is intended to be used for loading a given sector defined by the `WantedSector` type from `@reveal/cad-parsers` package.
+The result will be of type `ConsumedSector` which contains all the given geometry for the sector. 
+This package is responsible for fetching the sector (remote, local, cache, etc.) and feeding the data to a parser and packaging the result for usage in a scene.
+
+
+The facade of this package is as follows:
+```ts
+export interface SectorRepository {
+  loadSector(sector: WantedSector): Promise<ConsumedSector>;
+  clear(): void;
+}
+```
+
+The package contains two implementations for this facade: 
+- `V8SectorRepository`: i3d/f3d/ctm outputs
+- `GltfSectorRepository`: for Gltf-binary outputs
+
+
+Definitions for type dependencies from `@cad-parsers` are as follows:
+```ts
+export interface WantedSector {
+  modelIdentifier: string;
+  modelBaseUrl: string;
+  geometryClipBox: THREE.Box3 | null;
+  levelOfDetail: LevelOfDetail;
+  metadata: SectorMetadata;
+}
+```
+
+```ts
+export interface ConsumedSector {
+  modelIdentifier: string;
+  metadata: SectorMetadata;
+  levelOfDetail: LevelOfDetail;
+  group: AutoDisposeGroup | undefined;
+  instancedMeshes: InstancedMeshFile[] | undefined;
+}
+```

```

Comment:
This might quickly go stale, but big fan of a detailed description of the package. We could consider linking to files which is maybe a bit less brittle (but by no means guaranteed to remain stable either...)